ui/src/App.tsx

```diff
@@ -13,44 +15,61 @@ import CloudNav from 'src/pageLayout/components/CloudNav'
 import CloudOnly from 'src/shared/components/cloud/CloudOnly'
 
 // Types
-import {AppState} from 'src/types'
+import {AppState, CurrentPage, Theme} from 'src/types'
 
 interface StateProps {
   inPresentationMode: boolean
+  currentPage: CurrentPage
+  theme: Theme
 }
 interface OwnProps {
   children: ReactChildren
 }
 
-type Props = OwnProps & StateProps
-
-const App: SFC<Props> = ({children, inPresentationMode}) => (
-  <>
-    <CloudOnly>
-      <CloudNav />
-    </CloudOnly>
-    <AppWrapper presentationMode={inPresentationMode}>
-      <Notifications />
-      <TooltipPortal />
-      <NotesPortal />
-      <OverlayController />
-      <Nav />
-      {children}
-    </AppWrapper>
-  </>
-)
+type Props = OwnProps & StateProps & WithRouterProps
+
+const App: SFC<Props> = ({
+  children,
+  inPresentationMode,
+  currentPage,
+  theme,
+}) => {
+  const appWrapperClass = classnames('', {
+    'dashboard-light-mode': currentPage === 'dashboard' && theme === 'light',
+  })
+
+  return (
+    <>
+      <CloudOnly>
+        <CloudNav />
+      </CloudOnly>
+      <AppWrapper
+        presentationMode={inPresentationMode}
+        className={appWrapperClass}
+      >
+        <Notifications />
+        <TooltipPortal />
+        <NotesPortal />
+        <OverlayController />
+        <Nav />
+        {children}
+      </AppWrapper>
+    </>
+  )
+}
 
 const mstp = (state: AppState): StateProps => {
   const {
     app: {
       ephemeral: {inPresentationMode},
+      persisted: {currentPage, theme},
```

Comment:
I'm confused why we have a separate state for determining the current page- isn't it available via router path?

Reply:
@121watts talked to @ebb-tide about our approach here a bit, and we concluded that it's unnecessary to have `currentPage` stored in persisted state. What do you think about moving it to `ephemeral` or somewhere else?